src/table/components/withSelections.jsx

```diff
@@ -5,23 +5,30 @@ import { getSelectionStyle } from '../utils/styling-utils';
 
 export default function withSelections(CellComponent) {
   const HOC = (props) => {
-    const { selectionstate: selectionState, cell, selectiondispatch: selectionDispatch, styling, announce } = props;
+    const { selectionState, cell, selectionDispatch, styling, announce, column, ...passThroughProps } = props;
+
     const handleMouseUp = (evt) =>
       cell.isDim && evt.button === 0 && selectCell({ selectionState, cell, selectionDispatch, evt, announce });
+
     const selectionStyling = useMemo(
       () => getSelectionStyle(styling, cell, selectionState),
       [styling, cell, selectionState]
     );
 
-    return <CellComponent {...props} styling={selectionStyling} onMouseUp={handleMouseUp} />;
+    return <CellComponent {...passThroughProps} styling={selectionStyling} onMouseUp={handleMouseUp} />;
+  };
+
+  HOC.defaultProps = {
```

Comment:
Why does only this HOC need defualtProps?

Reply:
Only this HOC has not required propTypes property `column`

Reply:
I don't understand, we don't use it here, so why destruct it from `props`? Can't we just pass it through?

Reply:
The reason why we want to destruct it from props is that 
we do not want to pass the `column` to `withStyling`, as `column` is not needed in the DOM.
And When `hasColumnStyling` is true, `withColumnStyling` will not pass `column` to `withSelections`, 
so `column` could be null and should not be required.

Reply:
that makes sense